at_client/src/main/java/org/atsign/client/util/ConvertingUtils.java

```diff
@@ -0,0 +1,60 @@
+import java.io.*;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+
+// Main class
```

Comment:
Could you please give appropriate javadocs to the class and methods?

[Javadocs](https://www.baeldung.com/javadoc)